docs/user/dql/functions.rst

```diff
@@ -2194,6 +2197,7 @@ Description
 >>>>>>>>>>>
 
 Usage: week(date[, mode]) returns the week number for date. If the mode argument is omitted, the default mode 0 is used.
+If an argument of type `TIME` is given, the function will use the current date to calculate an output.
```

Comment:
```suggestion
If an argument of type `TIME` is given, the function will use the current date.
```

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/199/commits/eac4b4e2ca1fb8a3041a572b9915a9be212981e4